guillotina/api/service.py

```diff
@@ -36,6 +42,31 @@ def __setattr__(self, name, value):
 
 
 class Service(View):
+    __validator__ = None
+
+    async def validate(self):
+        data = await self.request.json()
+        if(self.__validator__ is None):
+            try:
+                requestBody = self.__config__['requestBody']
+                self.__schema__ = requestBody['content']['application/json']['schema']['$ref'].split('/')[-1]
+                self.__validator__ = get_schema_validator(self.__schema__)
+            except Exception as e:
```

Comment:
Maybe we only catch `(IndexError, KeyError)` here?

Reply:
yep, I will add validation for arbitrary schemas and add a method for getting validator.